README.md

```diff
@@ -1,2 +1,226 @@
-# cleanUI
- 
+# cleanUI - Component Library
+
+## What is cleanUI?
+
+**clean UI** is a lightweight and customizable Component Library.
+
+This library will make it simple for devs to use and style different CSS components!
+
+It helps you build your projects sooner
+
+## How to install cleanUI?
+
+To use the library just copy and paste the given code in the <head> part of the HTML. So that you can style your components by just adding class names to your HTML elements.
+
+```html
+<link rel="stylesheet" href="https://uiclean.netlify.app/Components/global.css" />
+```
+## Features
+
+1. **Open Source**
+
+2. **Responsive**
+
+3. **Clean code**
+
+4. **Ready to use**
+
+## Following are all the components that clean UI consist of: 
+
+1. [Typography](#typography)
+
+2. [Avatars](#avatars)
+
+3. [Alerts](#alerts)
+
+4. [Button](#buttons)
+
+5. [Badge](#badges)
+
+6. [Cards](#cards)
+
+7. [Input](#inputs)
+
+8. [Image](#images)
+
+9. [Lists](#lists)
+
+10. [Navigation](#navigation)
+
+11. [Modal](#modal)
+
+12. [Toast](#toast)
+
+13. [Rating](#rating)
```

Comment:
I would suggest to give links to the component in the documentation page, but its upto you, not a necessary change.